Podverse/PVMediaPlayer.swift

```diff
@@ -160,6 +160,10 @@ class PVMediaPlayer {
             PVDeleter.deleteEpisode(episodeId: episode.objectID, fileOnly: true)
             currentlyPlayingItem.didFinishPlaying = true
             playerHistoryManager.addOrUpdateItem(item: currentlyPlayingItem)
+            
+            if let topController = UIApplication.topViewController() as? MediaPlayerViewController {
```

Comment:
This is a sneaky idea and I do like sneaky but... PVMediaPlayer has a protocol and a delegate property. I think it would be better if we utilized that and set the PVMediaPlayerVC as the delegate of PVMediaPlayer and tell the MediaplayerVC to dismiss through a delegate method. This technique of getting the topVC is a lot of overhead that is unnecessary by extending the UIApplication.